app/src/main/res/values/integers.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <integer name= "edit_text_activity_reservation_ems_value">6</integer>
+    <integer name= "edit_text_number_main_activity_ems_value">5</integer>
+</resources>
```

Comment:
Add an empty line here to avoid this error

Reply:
Done